### PR TITLE
Add profil and  create

### PR DESCRIPTION
**Pull request by FaGru**

### CODE REVIEW

File: create.html [hunk 1]
```diff
@@ -7,30 +7,32 @@
...
+        <section class="create__section-wrap">
           <label for="add-answer">Add answer</label>
-          <input id="add-answer" type="text" />
+          <textarea id="add-answer" type="text" name="add-answer" cols="30" rows="5"></textarea>
         </section>
-        <section>
+        <section class="create__section-wrap">
           <label for="add-tags">Add tags (divided by comma) </label>
-          <input id="add-tags" type="text" />
+          <textarea name="add-tags" id="add-tags" cols="20" rows="3"></textarea>
```
Comment (JelenadlS) [inline create.html:24]: I git the hint from Michi to use here an `<input>` in order to have a smaller box - might be an idea for you as well 😊

File: images/no-bookmark.svg [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (JelenadlS) [inline images/no-bookmark.svg:1]: uh what is this? i do not really understand it 😅

File: styles/components/nav.css [hunk 3]
```diff
@@ -10,4 +10,10 @@
   border-top-right-radius: 1em;
   margin-top: 2em;
   padding: 1.5rem;
+  z-index: 2000;
```
Comment (JelenadlS) [inline styles/components/nav.css:13]: haha this is a quite high number 😄

File: styles/components/profil.css [hunk 4]
```diff
@@ -0,0 +1,39 @@
+.profil {
+  margin: 0;
```
Comment (JelenadlS) [inline styles/components/profil.css:2]: I saw this margin already quite often - maybe an idea to summarize it in utilities? 😊

**Review by JelenadlS — APPROVED** [review]
You are already so far, great job! 😊
I think overall it is looking very good! 👍🏼

File: images/no-bookmark.svg (hunk 2 above)
Comment (FaGru) [reply images/no-bookmark.svg:1]: i think this is from the bookmark svg 😅
I don't understand it too 🤷🏻

File: styles/components/nav.css (hunk 3 above)
Comment (FaGru) [reply styles/components/nav.css:13]: haha yeah the bookmark was in the front of the navbar, so i put this on 2000, that the navbar is ALWAYS in the front of everything... 😂

File: styles/components/profil.css (hunk 4 above)
Comment (FaGru) [reply styles/components/profil.css:2]: yes this is a good idea 👍🏻

File: create.html (hunk 1 above)
Comment (FaGru) [reply create.html:24]: first i had `<input>`, but it was to small and i liked the `<textarea>` with 3 rows more 🙂

**Pull request merged**